corehq/apps/userreports/tests/test_pillow.py

```diff
@@ -372,17 +379,24 @@ def test_bootstrap_if_needed(self, bootstrap_if_needed):
         bootstrap_if_needed.assert_called_once_with()
 
 
-class IndicatorPillowTest(TestCase):
+class IndicatorPillowTest(BaseRepeaterTest):
 
     @classmethod
     def setUpClass(cls):
         super(IndicatorPillowTest, cls).setUpClass()
         cls.config = get_sample_data_source()
+        cls.domain_obj.name = cls.config.domain
+        cls.domain_obj.save()
         cls.config.save()
```

Comment:
Instead of changing the domain name of `domain_obj` after it has been created, just override its name on the class, and it will be created by `super().setUpClass()` with the correct name.

```suggestion
class IndicatorPillowTest(BaseRepeaterTest):
    domain = 'user-reports'  # Domain name from sample data source

    @classmethod
    def setUpClass(cls):
        super(IndicatorPillowTest, cls).setUpClass()
        cls.config = get_sample_data_source()
        cls.config.save()
```

Reply:
https://github.com/dimagi/commcare-hq/pull/33781/commits/2d8752ed396d6ec49e6e3aeb8940ecc39f1ea970